OpenRA.Game/FieldLoader.cs

```diff
@@ -26,7 +25,6 @@ public static class FieldLoader
 	{
 		const char SplitComma = ',';
 
-		[Serializable]
```

Comment:
There's a couple of other `[Serializable]` attributes against Exception types that we may as well delete as well.